index.js

```diff
@@ -114,10 +120,10 @@ const runAction = () => {
 	}
 
 	log(`${release ? "Releasing" : "Building"} the Electron app…`);
-	run(
+	runIn(
 		`${useNpm ? "npx --no-install" : "yarn run"} electron-builder --${platform} ${
 			release ? "--publish always" : ""
-		}`,
+		}`, root
```

Comment:
Shouldn't the rest of the commands (e.g. `npm install`) also be run in that directory? If so, we could simply extend the `run` function with a new parameter instead of creating a new `runIn` helper.

Reply:
So it can depend on the kind of mono-repo setup you are running - I am not a monorepo expert or even an advocate. I believe I have what should be a common use case though. A mono-repo with separate packages for: the electron application, shared tooling, shared libraries, a web server (that uses the shared models), utilities, etc. The entire mono repo must be installed and built in order to build the main electron application.

Alternatively you could have a giant mono-repo with a lot of code that is not required for the electron application. That in my opinion would require a more custom build script than this generic use action.

Reply:
Ok, works for me. If somebody needs it, a second variable (e.g. `package_root`) could be introduced in the future, which would define where NPM/Yarn commands are run.